tests/e2e-kubernetes/scripts/run.sh

```diff
@@ -134,7 +147,8 @@ function create_cluster() {
       "$CI_ROLE_ARN" \
       "$INSTANCE_TYPE" \
       "$AMI_FAMILY" \
-      "$K8S_VERSION_EKSCTL"
+      "$K8S_VERSION_EKSCTL"\
+      "EKSCTL_PATCH_SELINUX_ENFORCING_FILE"
```

Comment:
Seems like a syntax error, this would be just interpreted as string without leading `$`
```suggestion
      "$EKSCTL_PATCH_SELINUX_ENFORCING_FILE"
```